yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -853,6 +856,39 @@ func fetchColumnsWithUnsupportedDataTypes() ([]utils.TableColumnsDataTypes, erro
 	return unsupportedDataTypes, nil
 }
 
+const ORACLE_PARTITION_DEFAULT_COLOCATION = `In case of Oracle, all partitions of a partitioned table are created as colocated by default. 
+The Assessment Report provides sharding/colocation recommendations for each partition individually.<br>
+However, due to certain limitations, these recommendations cannot be directly applied. 
+To manually modify the schema for sharding, please refer: <a class="highlight-link" href="https://github.com/yugabyte/yb-voyager/issues/1581">https://github.com/yugabyte/yb-voyager/issues/1581</a>.`
+
+func addNotesToAssessmentReport() {
+	log.Infof("adding notes to assessment report")
+	switch source.DBType {
+	case ORACLE:
+		assessmentReport.Notes = append(assessmentReport.Notes, ORACLE_PARTITION_DEFAULT_COLOCATION)
```

Comment:
let's only add this if there are actual partitioned tables in the schema.

Reply:
hmm makes sense. I will add a check based on whether a `partition.sql` file is present or not